README.md

```
@@ -62,6 +62,54 @@ docker images
 ```bash
 docker image prune
 ```
+---
+### 데이터 확인 및 관리
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)